clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/agent/CleanupAlarmsAgent.groovy

```diff
@@ -34,12 +34,17 @@ import groovy.util.logging.Slf4j
 import org.joda.time.DateTime
 
 import java.util.concurrent.TimeUnit
+import java.util.regex.Pattern
 
 @Slf4j
 class CleanupAlarmsAgent implements RunnableAgent, CustomScheduledAgent {
   public static final long POLL_INTERVAL_MILLIS = TimeUnit.HOURS.toMillis(24)
   public static final long DEFAULT_TIMEOUT_MILLIS = TimeUnit.MINUTES.toMillis(20)
 
+  private static String uuidPattern = "[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}"
+
+  public static final Pattern ALARM_NAME_PATTERN = Pattern.compile("(.+-)+v[0-9]{3}-alarm-${uuidPattern}")
```

Comment:
Probably. I'm overreacting a bit. And you're right, the grouping isn't important.